jina/serve/runtimes/gateway/request_handling.py

```diff
@@ -61,17 +62,35 @@ def __init__(
                 labelnames=('runtime_name',),
             ).labels(runtime_name)
 
+            self._failed_requests_metrics = Gauge(
```

Comment:
I believe this should be a Counter and not a Gauge since the number cannot decrease